lib/rules/no-pause-test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @fileoverview Disallow usage of pauseTest() in tests
+ * @author Scott Kidder
```

Comment:
I would suggest not including this comment. None of our other files have author comments.

Reply:
:+1: Thanks, yeoman generated this and I should've noticed.